js_modules/dagster-ui/packages/ui-core/src/runs/LogsProvider.tsx

```diff
@@ -293,6 +293,7 @@ const LogsProviderWithQuery = (props: LogsProviderWithQueryProps) => {
     RUN_LOGS_QUERY,
     {
       notifyOnNetworkStatusChange: true,
+      fetchPolicy: 'no-cache',
```

Comment:
Ahh, is Apollo cache slowing us down here?

Reply:
Yeah and this also brings it inline with our subscription variant which already `no-cache`